haystack/agents/base.py

```diff
@@ -24,16 +24,17 @@
 
 class Tool:
     """
-    A tool is a pipeline or node that also has a name and description. When you add a Tool to an Agent, the Agent can
-    invoke the underlying pipeline or node to answer questions. The wording of the description is important for the
-    Agent to decide which tool is most useful for a given question.
+    Agent uses tools to find the best answer. A tool is a pipeline or a node. When you add a tool to an Agent, the Agent can
+    invoke the underlying pipeline or node to answer questions. 
+    
+    You must provide a name and a description for each tool. The name should be short and should indicate what the tool can do. The description should explain what the tool is useful for. The Agent uses the description to decide when to use a tool, so the wording you use is important.
 
     :param name: The name of the tool. The Agent uses this name to refer to the tool in the text the Agent generates.
-        The name should be short, ideally one token, and a good description of what the tool can do, for example
+        The name should be short, ideally one token, and a good description of what the tool can do, for example:
```

Comment:
Note that Malte suggested a change to this exact line in a PR that is still pending https://github.com/deepset-ai/haystack/pull/4269/files